Makefile

```diff
@@ -18,7 +18,7 @@ all: test build
 test: schema-check cover.out
 
 schema-check:
-	go test -v ./test/schema
+	go run ./cmd/validator/main.go ./deploy/chart/catalog_resources
```

Comment:
`go run validator` accepts a path to a directory of unique catalogs or packages, but not versions of the same catalog.